components/docs-chef-io/content/automate/ha_existing_a2ha_to_automate_ha.md

```diff
@@ -27,9 +27,33 @@ This page explains migrating the existing A2HA data to the newly deployed Chef A
 ## Prerequisites
 
 - Ability to mount the file system, which was mounted to A2HA Cluster for backup purposes, to Automate HA.
-
 - Configure the A2HA to take backup on a mounted network drive (location example: `/mnt/automate_backup`).
 
+## Capture information about the current A2HA instance
+
+In order to verify the migration is completed successfully we'll need to capture some information about the current installation. The following script will capture counts of objects in the Chef-Infra Server that we can compare with the server after the migration has been completed.
```

Comment:
Replace it with: We'll need to capture information about the current installation to verify the migration is completed successfully. The following script will capture counts of objects in the Chef-Infra Server that we can compare with the server after the migration has been completed.